shortfin/python/shortfin_apps/llm/components/kvcache/base_attention_cache.py

```diff
@@ -57,24 +139,9 @@ def acquire_pages_for_tokens(
         pages_needed = math.ceil(token_count / self.tokens_per_page)
         pages = self.page_pool.acquire_free_pages(pages_needed)
 
-        n_cached_tokens = 0
-
-        return pages, n_cached_tokens
+        if pages is None:
+            raise CacheAllocationFailure()
 
-    def publish_pages(self, tokens, pages) -> None:
-        """
-        Given a list of tokens and pages containing KV corresponding to these tokens, make these pages available to other requests.
-
-        Associates the tokens with the pages, and mark them as done writing.
-
-        It is assumed that hereafter, the calling request will not modify these pages, at least not the positions [0:len(tokens)].
-        """
-
-        pass  # the base implementation doesn't cache unfinished requests.
+        n_cached_tokens = 0
```

Comment:
Looks like this variable isn't used anymore

Reply:
fixd! ty